test/e2e/sample_e2e_test.go

```diff
@@ -1,12 +1,33 @@
 package e2e
 
 import (
+	"fmt"
+
 	"github.com/blang/semver/v4"
 	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/clientset/versioned"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/operatorclient"
 	cu "github.com/operator-framework/operator-lifecycle-manager/test/e2e/catalogutil"
 )
 
 var _ = Describe("Sample", func() {
+
+	var (
+		c   operatorclient.ClientInterface
+		crc versioned.Interface
+	)
+	BeforeEach(func() {
+		c = newKubeClient()
+		crc = newCRClient()
+	})
+
+	AfterEach(func() {
+		TearDown(testNamespace)
```

Comment:
I am not entirely sure if I need to this, wondering if this would affect other test runs

Reply:
I searched through the code looking for `Namespaces().Create` and none of those refs seem to create the `testNamespace`. It's not clear to me how this gets re-created when its torn down. I think we'd need to ask in `#olm-dev` channel how this is handled.